src/Element/common.c

```diff
@@ -0,0 +1,66 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   common.c                                           :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: minjungk <minjungk@student.42seoul.>       +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2023/05/31 04:09:51 by minjungk          #+#    #+#             */
+/*   Updated: 2023/06/03 20:44:48 by minjungk         ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "common.h"
+
+static char	*next_rgb(int *buf, char *curr)
+{
+	if (curr == NULL || ft_isdigit(curr[0]) == 0)
+		return (NULL);
+	*buf = ft_atoi(curr);
+	if (!(0 <= *buf && *buf <= 255))
+		return (NULL);
+	while (ft_isdigit(curr[0]))
+		++curr;
+	if (curr[0] != '\0' && curr[0] != '\r' && curr[0] != ',')
```

Comment:
개행`\n`에 대한 처리가 필요합니다.
```suggestion
	if (curr[0] != '\0' && ft_strchr(",\r\n", curr[0]) == NULL)
```